canonical-web-and-design/ubuntu.com
# Ubuntu Masters conference: phase 3

## Done

- Updated /masters-conference
- Deleted Brendan Gregg and Joe Sandoval pages per [brief](https://docs.google.com/document/d/1tj_XEsP0PvgpfLNb4zANmM73TwV4rF-9I1etzODRsfQ/edit)

**The brief specifies to not make this live until the 10th of Feb**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/masters-conference
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1Z63eMCe9o4Onk2ezPFLPDTbI-uJezAOzI-aSpXmZ5lc/edit#)


## Issue / Card

Fixes #6520
